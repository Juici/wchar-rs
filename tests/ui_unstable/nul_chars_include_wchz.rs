use wchar::{include_wchz, wchar_t};

const NUL_CHARS: &[wchar_t] = include_wchz!("../data/nul_chars.txt");

fn main() {}
