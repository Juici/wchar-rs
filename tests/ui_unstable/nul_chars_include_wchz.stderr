error: C-style string cannot contain nul characters
 --> $DIR/nul_chars_include_wchz.rs:3:45
  |
3 | const NUL_CHARS: &[wchar_t] = include_wchz!("../data/nul_chars.txt");
  |                                             ^^^^^^^^^^^^^^^^^^^^^^^
